google/cloud/bigquery/v2/minimal/internal/job_client_test.cc

```diff
@@ -31,30 +32,30 @@ using ::google::cloud::testing_util::StatusIs;
 using ::testing::HasSubstr;
 using ::testing::Return;
 
+Job GetJob() {
+  Job job;
+  job.etag = "jtag";
+  job.kind = "jkind";
+  job.id = "j123";
+  job.self_link = "jselfLink";
+  job.user_email = "juserEmail";
+  job.status.state = "DONE";
+  job.reference.project_id = "p123";
+  job.reference.job_id = "j123";
+  job.configuration.job_type = "QUERY";
+  job.configuration.query_config.query = "select 1;";
+  return job;
+}
+
 TEST(JobClientTest, GetJobSuccess) {
   auto mock_job_connection = std::make_shared<MockBigQueryJobConnection>();
 
-  std::string expected_payload =
-      R"({"kind": "jkind",
-          "etag": "jtag",
-          "id": "j123",
-          "self_link": "jselfLink",
-          "user_email": "juserEmail",
-          "status": {"state": "DONE"},
-          "reference": {"project_id": "p123", "job_id": "j123"},
-          "configuration": {
-            "job_type": "QUERY",
-            "query_config": {"query": "select 1;"}
-          }})";
   EXPECT_CALL(*mock_job_connection, GetJob)
       .WillOnce([&](GetJobRequest const& request) {
         EXPECT_EQ("test-project-id", request.project_id());
         EXPECT_EQ("test-job-id", request.job_id());
-        BigQueryHttpResponse http_response;
-        http_response.payload = expected_payload;
-        auto const& job_response =
-            GetJobResponse::BuildFromHttpResponse(http_response);
-        return make_status_or(job_response->job);
+        auto job = GetJob();
+        return make_status_or(job);
```

Comment:
nit: can we just `return make_status_or(GetJob());`? (or `MakeTestJob()` if you take my other suggestion)